ci-operator/step-registry/openshift-qe/installer/bm/deploy/openshift-qe-installer-bm-deploy-commands.sh

```diff
@@ -91,6 +86,10 @@ ssh ${SSH_ARGS} root@${bastion} "
    ansible -i ansible/inventory/$LAB_CLOUD.local bastion -m script -a /root/clean-resources.sh
    ansible-playbook -i ansible/inventory/$LAB_CLOUD.local ansible/setup-bastion.yml | tee /tmp/ansible-setup-bastion-$(date +%s)
    ansible-playbook -i ansible/inventory/$LAB_CLOUD.local ansible/${TYPE}-deploy.yml -v | tee /tmp/ansible-${TYPE}-deploy-$(date +%s)
+   mkdir -p /root/${TYPE}/$LAB_CLOUD
+   ansible -i ansible/inventory/$LAB_CLOUD.local bastion -m copy -a remote_src=/root/${TYPE}/kubeconfig dest=/root/$TYPE/$LAB_CLOUD/kubeconfig
    deactivate
    rm -rf .ansible
 "
+
+scp -q ${SSH_ARGS} root@${bastion}:/root/${TYPE}/${LAB_CLOUD}/kubeconfig ${SHARED_DIR}/kubeconfig
```

Comment:
I would suggest **_/root/$lab/$lab_cloud/$type/kubeconfig_** instead, otherwise when we start moving into the scalelab we can have folder collisions.

Reply:
you're right, fixed